src/controllers/shared.controllers/director.correspondence.address.link.controller.ts

```diff
@@ -33,7 +36,7 @@ export const getCorrespondenceLink = async (req: Request, res: Response, next: N
   }
 };
 
-export const postCorrespondenceLink = async (req: Request, res: Response, next: NextFunction, templateName: string, nextPageOnYesUrl: string, nextPageOnNoUrl: string, backUrlPath: string) => {
+export const postCorrespondenceLink = async (req: Request, res: Response, next: NextFunction, templateName: string, nextPageUrl: any, backUrlPath: string, isUpdate: boolean) => {
```

Comment:
Possible to define a type for nextPageUrl instead of any. A strict tsconfig change in the future might break it.
maybe something like interface CorrespondenceNextPage: {yes: string, no: string}